doc/conf.py

```diff
@@ -104,3 +104,7 @@
 
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'resticdoc'
+
+extlinks = {
+    'issue': ('https://github.com/restic/restic/issues/%s', '#'),
```

Comment:
Nice, I didn't know that this is possible. Is it compatible with readthedocs?

Reply:
yes, it is.